collector/perfcounter.go

```diff
@@ -0,0 +1,54 @@
+package collector
+
+import (
+	"github.com/pegasus-kv/meta-proxy/config"
+	"github.com/sirupsen/logrus"
+)
+
+var pfcType string
+
+var (
+	TableWatcherEvictCounter Counter
+	ClientConnectionCounter  Counter
+)
+
+type Counter interface {
+	Add(value int64)
+	Incr()
+}
+
+func InitPerfCounter() {
+	pfcType = config.Cfg.Pfc.Type
```

Comment:
```suggestion
	perfCounterType = config.Cfg.PerfCounter.Type
```

Stop using abbreviation everywhere